boards/others/esp32c3_supermini/esp32c3_supermini.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 Arrel Neumiller
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: esp32c3_supermini
+name: ESP32-C3-SUPERMINI
+type: mcu
+arch: riscv
+toolchain:
+  - zephyr
+supported:
+  - adc
+  - gpio
+  - i2c
+  - watchdog
+  - uart
+  - dma
+  - pwm
+  - spi
+  - counter
+  - entropy
+testing:
+  ignore_tags:
+    - net
+    - bluetooth
```

Comment:
I'm curious: why do these need to be explicitly ignored? You're not listing these features under `supported:`, so they shouldn't get picked up by any test filters either way.

Reply:
I assumed this was necessary based on the examples of other in-tree Espressif boards as the BT and WiFi binaries are provided externally and are not within the scope of Zephyr testing.

Reply:
My guess is that it's not necessary for those either (it might have been a long time ago, not sure, but I have a feeling that's not the case any longer). This is how unnecessary code and complexity persists and grows in a big project, when code just gets copied around without anyone actually stopping to question is it's correct or needed to begin with.

Reply:
Thank you for the feedback, @jhedberg. To ensure we address this properly, I propose filing a bug report to investigate whether explicitly ignoring these features is still necessary. This will help us determine if any adjustments are needed to simplify the code.

Reply:
I was thinking that you could just take the more empirical approach, i.e. remove these lines and see if CI still gives you all green.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/8078
@jhedberg I don't know for certain that it's effectively not needed anymore or does nothing, but the idea at least is that it's to prevent networking and bt tests to run on ESP32 platforms in CI since they would not compile without the esp32 blobs (that is, until PR above is merged)

cc @sylvioalves

Reply:
@kartben @sylvioalves thanks for the references. What confuses me is that we have the "supported" field in the board metadata. Doesn't it sound like a bug if CI would attempt to build features which are not explicitly listed in this field?

Reply:
> I was thinking that you could just take the more empirical approach, i.e. remove these lines and see if CI still gives you all green.
@jhedberg I'll consider that approach next time. Pre-CI habit on my part i.e. "Whatever you do, don't break the build!"

Reply:
@rlneumiller you can remove both entries as there is no issue at all:
https://github.com/zephyrproject-rtos/zephyr/pull/82368